app/src/components/EditEvent/EditEvent/EditEvent.tsx

```diff
@@ -37,7 +37,9 @@ export const EditEvent = ({ eventResult: event, updateEvent }: IProps) => {
     event.start.date !== event.end.date
   );
 
-  const [hasShortname, _setHasShortname] = useState(false);
+  const [hasShortname, _setHasShortname] = useState(
+    event.shortname !== undefined
```

Comment:
Modelerer vi ikke litt for mye ting med undefined? Kunne man ikke utnyttet Typescript litt bedre her?

Reply:
Ja du sier noe der 🤔 Dette kommer av at vi bruker `option` backend, men vi kunne jo brukt en rikere type i redigeringslogikken frontend.